apps/src/lab2/views/components/Instructions.tsx

```diff
@@ -233,15 +261,27 @@ const InstructionsPanel: React.FunctionComponent<InstructionsPanelProps> = ({
                   handleInstructionsTextClick={handleInstructionsTextClick}
                 />
               )}
-              {canShowNextButton && (
-                <button
-                  id="instructions-feedback-button"
-                  type="button"
+              {canShowContinueButton && (
+                <Button
+                  text={commonI18n.continue()}
+                  disabled={isFinished}
                   onClick={onNextPanel}
-                  className={moduleStyles.buttonNext}
-                >
-                  {commonI18n.continue()}
-                </button>
+                  className={moduleStyles.button}
+                />
+              )}
+              {canShowFinishButton && (
+                <Button
+                  text={commonI18n.finish()}
+                  disabled={isFinished}
+                  onClick={onFinish}
+                  className={moduleStyles.button}
+                />
+              )}
+              {finalMessage && isFinished && (
+                <EnhancedSafeMarkdown
```

Comment:
Out of curiosity, why do we need to use markdown here (instead of just CSS-styled text)?

Reply:
Thanks for the question! I was copying what was done with  `message` above but on second thought, I'll use a `Typography` element.